--- app.py
+++ app.py
@@ -6,3 +6,3 @@
-# from graphene import Schema
-# from virga.plugins.graphql import GraphQLRoute
-# from .gql import RootQuery
+from graphene import Schema
+from virga.plugins.graphql import GraphQLRoute
+from .gql import RootQuery
@@ -42,6 +42,6 @@
-# # Mounts a Graphene executor with schema to '/graphql'. POST requests to
-# # the route get executed while GET requests will render GraphiQL.
-# # GraphQLRoute accepts any Graphene Schema object.
-# #
-# # Read more: https://docs.graphene-python.org/en/stable/types/schema/
-# app.add_route("/graphql", GraphQLRoute(schema=Schema(query=RootQuery)))
+# Mounts a Graphene executor with schema to '/graphql'. POST requests to
+# the route get executed while GET requests will render GraphiQL.
+# GraphQLRoute accepts any Graphene Schema object.
+#
+# Read more: https://docs.graphene-python.org/en/stable/types/schema/
+app.add_route("/graphql", GraphQLRoute(schema=Schema(query=RootQuery)))
