--- app.py
+++ app.py
@@ -10,4 +10,4 @@
-# from sqlalchemy.ext.asyncio import AsyncSession
-# from sqlalchemy import select
-# from .database import async_session
-# from .database.models import ExampleUser
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy import select
+from .database import async_session
+from .database.models import ExampleUser
@@ -31,9 +31,9 @@
-# # Spawn an asynchronous database session and fetch a database object.
-# # https://docs.sqlalchemy.org/en/14/orm/session_basics.html
-# # https://docs.sqlalchemy.org/en/14/orm/session_basics.html#querying-2-0-style
-# @app.get("/db")
-# async def db_home(session: AsyncSession = Depends(async_session)):
-#     stmt = select(ExampleUser).filter_by(id=1)
-#     result = await session.execute(stmt)
-#     result = result.scalars().first()
-#     return {"message": f"Hello, {result.name}!"}
+# Spawn an asynchronous database session and fetch a database object.
+# https://docs.sqlalchemy.org/en/14/orm/session_basics.html
+# https://docs.sqlalchemy.org/en/14/orm/session_basics.html#querying-2-0-style
+@app.get("/db")
+async def db_home(session: AsyncSession = Depends(async_session)):
+    stmt = select(ExampleUser).filter_by(id=1)
+    result = await session.execute(stmt)
+    result = result.scalars().first()
+    return {"message": f"Hello, {result.name}!"}
