--- app.py
+++ app.py
@@ -12,4 +12,4 @@
-# from sqlalchemy.ext.asyncio import AsyncSession
-# from sqlalchemy import select
-# from .database import async_session
-# from .database.models import Widget
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy import select
+from .database import async_session
+from .database.models import Widget
@@ -53,9 +53,9 @@
-# # Spawn an asynchronous database session and fetch a database object.
-# # https://docs.sqlalchemy.org/en/14/orm/session_basics.html
-# # https://docs.sqlalchemy.org/en/14/orm/session_basics.html#querying-2-0-style
-# @app.get("/db")
-# async def db_home(session: AsyncSession = Depends(async_session)):
-#     stmt = select(Widget).where(Widget.id == 1)
-#     result = await session.execute(stmt)
-#     result = result.scalar_one()
-#     return {"message": f"This is a {result.name} widget!"}
+# Spawn an asynchronous database session and fetch a database object.
+# https://docs.sqlalchemy.org/en/14/orm/session_basics.html
+# https://docs.sqlalchemy.org/en/14/orm/session_basics.html#querying-2-0-style
+@app.get("/db")
+async def db_home(session: AsyncSession = Depends(async_session)):
+    stmt = select(Widget).where(Widget.id == 1)
+    result = await session.execute(stmt)
+    result = result.scalar_one()
+    return {"message": f"This is a {result.name} widget!"}
