--- settings.py
+++ settings.py
@@ -3,1 +3,1 @@
-# from functools import lru_cache
+from functools import lru_cache
@@ -15,19 +15,19 @@
-    # db_driver: str = "postgresql+asyncpg"
-    # postgres_host: str = "localhost"
-    # postgres_port: int = 5432
-    # postgres_user: str
-    # postgres_password: str
-    # postgres_db: str
-
-    # @property
-    # @lru_cache(maxsize=None)
-    # def db_url(self):
-    #     # dialect+driver://username:password@host:port/database
-    #     return "{}://{}:{}@{}:{}/{}".format(
-    #         self.db_driver,
-    #         self.postgres_user,
-    #         self.postgres_password,
-    #         self.postgres_host,
-    #         self.postgres_port,
-    #         self.postgres_db,
-    #     )
+    db_driver: str = "postgresql+asyncpg"
+    postgres_host: str = "localhost"
+    postgres_port: int = 5432
+    postgres_user: str
+    postgres_password: str
+    postgres_db: str
+
+    @property
+    @lru_cache(maxsize=None)
+    def db_url(self):
+        # dialect+driver://username:password@host:port/database
+        return "{}://{}:{}@{}:{}/{}".format(
+            self.db_driver,
+            self.postgres_user,
+            self.postgres_password,
+            self.postgres_host,
+            self.postgres_port,
+            self.postgres_db,
+        )
